src/configure/configurers/azurePipelineConfigurer.ts

```diff
@@ -127,6 +159,62 @@ export class AzurePipelineConfigurer implements Configurer {
         return path.join(inputs.sourceRepository.localPath, await LocalGitRepoHelper.GetAvailableFileName('azure-pipelines.yml', inputs.sourceRepository.localPath));
     }
 
+    public async checkInPipelineFileToRepository(inputs: WizardInputs, localGitRepoHelper: LocalGitRepoHelper): Promise<string> {
+
+        let commitMessage: string;
+        let initializeGitRepository = !inputs.sourceRepository.remoteUrl;
+
+        if (!inputs.sourceRepository.remoteUrl) {
+            let repositoryName = path.basename(inputs.sourceRepository.localPath).trim().replace(/[^a-zA-Z0-9-]/g, '');
+            let repository = await this.azureDevOpsClient.createRepository(inputs.organizationName, inputs.project.id, repositoryName);
```

Comment:
Should repository name be just project name itself, as we have for DevOps project creation

Reply:
We actually set repository name as AzurePipelines-{repo_name}. However in this case we do not have repo name, all we have is workspace folder name. Will work with PM for the names

Reply:
@pulkitaggarwl Can you suggest default name for the repository ? Also, should there be any parity between project and repo name ?

Reply:
Spec: repository name will be folder name of the workspace, if it has only special characters then default name is codetoazure.